.ci/downstream.stages

```diff
@@ -1,13 +1,16 @@
 def call(def propertiesFolderPath) {
-    stage('Downstream Build') {
+    stage('Full Downstream Build') {
         final REPOSITORY_LIST_FILE = "./script/repository-list.txt"
         final SETTINGS_XML_ID = "771ff52a-a8b4-40e6-9b22-d54c7314aa1e"
+        final PROPERTIES_FILE_PATH = "${propertiesFolderPath}/fullDownstreamBuild.properties"
```

Comment:
either we rename the `downstream.stages` to `fullDownstream.stages` or we do it for `fullDownstreamBuild.properties` to `downstreamBuild.properties`, otherwise it looks a bit messy and can confuse people. wdyt?

Reply:
@Ginxo DONE